### ISSUE TITLE
Rancher CLI - muliclusterapp name is not seen on installing an app

### ISSUE THREAD

**Issue by sowmyav27** [rancher/rancher#22527]
**What kind of request is this (question/bug/enhancement/feature request):** bug

**Steps to reproduce (least amount of steps as possible):**
on running command `rancher multiclusterapps install sentry `, no app name is seen on the cli ui.
```
$ ./rancher multiclusterapps install sentry 
Installing multi-cluster app ""...
```

**Expected Result:**
Deployed Multi cluster App name be displayed

**Other details that may be helpful:**

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): rancher cli - `v2.3.0-rc6`
- Installation option (single install/HA): single

**Comment by cjellick** [comment]
moving out

**Comment by deniseschannon** [comment]
available in CLI v2.3.0-rc7

**Comment by sowmyav27** [comment]
Verified with cli `v2.3.0-rc7`
Deployed Multi cluster App name is displayed
```
./rancher multiclusterapps install sentry 
Installing multi-cluster app "mca-z2srx"...
```
